src/services/CreateAnalysisService.tsx

```diff
@@ -62,26 +63,29 @@ class CreateAnalysisService {
     return imgs.filter((img: DcmImage) => this.isImgSelected(selectedStudyUIDs, img));
   }
 
+  /**
+   * Runs pl-med2img and the pl-covidnet/pl-ct-covidnet given the pl-dircopy instances and DcmImgages generated from copyFiles
+   * @param XrayModel 
+   * @param CTModel 
+   * @returns 
+   */
```

Comment:
Outdated docstring

I believe this docstring was added before my feedback about not needing to separate pl-dircopy plugin runs from other plugin runs.

I suggest either updating the docstring (ensure it follows JSDoc standards) or removing this docstring.